config/settings.yml

```diff
@@ -989,6 +989,8 @@
     :keep_reports: 6.months
     :purge_window_size: 100
   :queue_timeout: 1.hour
+  :session_log_threshold: 100.kilobytes
+  :session_element_threshold: 10.kilobytes
```

Comment:
I think this should go in the `session` section (not reporting), below, then we can remove the `session_` prefix:

(line 1046 below)
```
 :session:
   :element_threshold: 10.kilobytes
   :interval: 60
   :log_threshold: 100.kilobytes
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1 -I 1M"
   :show_login_info: true
   :timeout: 3600
```